### DISCUSSION TITLE
Access Code

### DISCUSSION THREAD

**Discussion by Maxime0506** [rednblkx/HomeKey-ESP32#88] [Q&A]
Hello I wanted to ask if it is possible to add the access code feature to the lock so it can be opened with a keypad and the code can be changed in the home app

**Comment by rednblkx** [comment]
Hi! Access Codes is something I’ve considered doing but is not a priority so don’t know when I’ll be getting to that.

**Reply by Dragonpark** [reply]
Access codes would be a great addition to this project. I'm currently trying to cobble something together, but it requires two ESPs and somehow managing the access codes separately.
